analyzers/its/expected/Automapper/AutoMapper--net461-S3260.json

```diff
@@ -0,0 +1,17 @@
+{
+"issues":  [
+{
+"id":  "S3260",
+"message":  "Private classes or records which are not derived in the current assembly should be marked as 'sealed'.",
```

Comment:
Please, do not forget to update the rspec too. (https://sonarsource.github.io/rspec/#/rspec/S3260/csharp)

Reply:
https://github.com/SonarSource/rspec/pull/378

Reply:
The linked PR looks good. It's now merged.